src/Mod/PartDesign/Gui/TaskBooleanParameters.cpp

```diff
@@ -89,10 +89,9 @@ TaskBooleanParameters::TaskBooleanParameters(ViewProviderBoolean* BooleanView, Q
         auto shortcut = rcCmdMgr.getCommandByName("Std_Delete")->getShortcut();
         action->setShortcut(QKeySequence(shortcut));
     }
-#if QT_VERSION >= QT_VERSION_CHECK(5, 10, 0)
+
     // display shortcut behind the context menu entry
     action->setShortcutVisibleInContextMenu(true);
-#endif

```

Comment:
There seem to be a lot of these "display shortcut" related code.
I think all of them would be easier to read if you have a new line afterwards.

The old `#endif` line did kind of work like a new line to separate the lines of code.